superset-frontend/plugins/plugin-chart-echarts/package.json

```diff
@@ -24,10 +24,10 @@
     "lib"
   ],
   "dependencies": {
+    "@types/react-redux": "^7.1.10",
     "d3-array": "^1.2.0",
-    "lodash": "^4.17.21",
     "dayjs": "^1.11.13",
-    "@types/react-redux": "^7.1.10"
+    "lodash": "^4.17.21"
```

Comment:
should this be a peerDep?

Reply:
@mistercrunch hmm not sure. Since we changed the approach in the PR to use `Intl.Collator` there's no real changes in `package.json` -- it's just sorting these.

Reply:
oh wait, I thought it was added, but you simply re-ordered  them. Now wondering whether it should have been a peerDep in the first place (?)

Reply:
yeah, they were just re-ordered by running `pre-commit` before pushing this. I'm not too familiar with the `package.json` configs so not sure if we need these explicitly listed as a `dependency`, but I can look into updating this if needed.